crypto/encryption.go

```diff
@@ -3,13 +3,18 @@ package crypto
 import "github.com/ProtonMail/go-crypto/openpgp/packet"
 
 type EncryptionProfile interface {
-	EncryptionConfig() *packet.Config
+	EncryptionConfig(messageSizeHint uint64) *packet.Config
```

Comment:
We might need more options here in the future without breaking backward compatibility.
What about introducing a `struct EncHandleSettings`  and passing `&encHandleSettings` instead with default nil?